src/components/callout/callout.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import Callout from './callout';
+import ShootingStar from '../../illustrations/shooting-star.svg';
+import ConstellationBigDipper from '../../illustrations/constellation-big-dipper.svg';
+
+const EXAMPLE_ILLUSTRATIONS = {
+  None: null,
+  ShootingStar: <ShootingStar />,
+  ConstellationBigDipper: <ConstellationBigDipper />,
+};
+
+export default {
+  component: Callout,
+  title: 'Components/Callout',
+  parameters: {
+    componentSubtitle:
+      'Display additional information detached from your surrounding text content',
+  },
+  argTypes: {
+    illustration: {
+      options: Object.keys(EXAMPLE_ILLUSTRATIONS),
+      mapping: EXAMPLE_ILLUSTRATIONS,
+      description:
+        'This is an example list of icons, there is not default set. You have to provide a svg react component/',
+      control: {
+        type: 'radio', // Type 'select' is automatically inferred when 'options' is defined
+        labels: {
+          // 'labels' maps option values to string labels
+          None: 'None',
+          ShootingStar: 'Shooting Star',
+          ConstellationBigDipper: 'Constellation (Big Dipper)',
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof Callout>;
+
+const Template: ComponentStory<typeof Callout> = (args) => {
+  return <Callout {...args}>{args.children}</Callout>;
+};
+
+export const Regular = Template.bind({});
+Regular.args = {
+  illustration: <ConstellationBigDipper />,
```

Comment:
For some reason this does not work. It shows no illustration here without explicitly selecting one. It does work for the `WithLongText` story but not for `Regular`. I can't find any difference... maybe you can figure this out? Of course I don't consider this a blocker... although I'd love to understand what's going on here :thinking:

Reply:
I have updated the args definition. It's now working properly since I switched to a proper args mapping but I still have big problems selecting the default value from the list. The documentation is no help and I guess reading the sources will help. Anyway it's working now.